src/components/Search/Search.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import "react-bootstrap-typeahead/css/Typeahead.css";
+import "react-bootstrap-typeahead/css/Typeahead-bs4.css";
+import Button from "react-bootstrap/Button";
+import { Typeahead } from "react-bootstrap-typeahead";
+import FormGroup from "react-bootstrap/FormGroup";
+import InputGroup from "react-bootstrap/InputGroup";
+
+class Search extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      search: "",
+      cityNames: []
+    };
+  }
+
+  onSearch = () => {
+    if (!this.state.search) {
+      return;
+    }
+
+    const url = this.state.search.href;
+    const slugIndex = url.lastIndexOf(":") + 1;
+    const slug = url.substr(slugIndex).slice(0, -1);
+    console.log(slug);
+  };
+
+  componentDidMount() {
+    fetch("https://api.teleport.org/api/urban_areas/")
+      .then(response => response.json())
+      .then(response => {
+        console.log(response);
```

Comment:
There should be no debug `console.log` calls when you commit.
There are a few `console.log` that can be removed from this file.